4.0.4 broken with TypeScript strict mode

Trying to import v4.0.4 in a project that has TypeScript strict mode enabled:
```
import * as PIXISound from '@pixi/sound';
```
causes errors like this:
```
node_modules/@pixi/sound/dist/pixi-sound.d.ts:384:5 - error TS2416: Property 'init' in type 'HTMLAudioInstance' is not assignable to the same property in base type 'IMediaInstance'.
  Type '(media: HTMLAudioMedia) => void' is not assignable to type '(parent: IMedia) => void'.
    Types of parameters 'media' and 'parent' are incompatible.
      Type 'IMedia' is missing the following properties from type 'HTMLAudioMedia': parent, _source, source, eventNames, and 9 more.

384     init(media: HTMLAudioMedia): void;
        ~~~~
```
The errors look a bit mysterious but I traced it down to the more strict event typings added in a0523318fc1c1c130da7c98f699e24ab8c13a948.

Just to be clear: v4.0.3 works with strict mode

--- a/src/interfaces/IMediaInstance.ts
+++ b/src/interfaces/IMediaInstance.ts
@@ -110,8 +110,7 @@ interface IMediaInstance
     on(event: 'progress', fn: (progress: number, duration: number) => void, context?: any): this;
     on(event: 'resumed' | 'paused' | 'start' | 'end' | 'stop', fn: () => void, context?: any): this;
     off(event: 'resumed' | 'paused' | 'start' | 'end' | 'progress' | 'pause' | 'stop',
-        // eslint-disable-next-line @typescript-eslint/ban-types
-        fn: Function, context?: any, once?: boolean): this;
+        fn?: (...args: any[]) => void, context?: any, once?: boolean): this;
 
     /**
      * Fired when the sound when progress updates.

